### ISSUE TITLE
1.8% Spell Crit reduction does not appear to be applied on live

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#873]
Originally reported on Google Code with ID 874

```
What steps will reproduce the problem?
1. Equip with less than 1.8%, 1.7%, 1.6% Spell Crit (ICC gear or less works).
2. Attack the boss target dummy

What is the expected output? What do you see instead?

You should not be seeing any crits happening, Yet crits are taking place

What version of the product are you using? On what operating system?

SimC 4.2.2 Windows 64

Please provide any additional information below.

I explained what happened here:

http://elitistjerks.com/f75/t110187-cataclysm_mage_simulators_formulators/p16/#post2003002

Basically I spent about an hour and a half or so of casting Ice Lance on the boss target
dummy with less than 1.8% crit rating. I had 1 crit happen at 1.77% crit, 1 crit at
1.68%, and; after 500+ Ice Lances; 1 crit at 1.59% crit rating.
```

Reported by `tseltman` on 2011-09-23 18:53:18
